antora/ec-policies-antora-extension/index.js

```diff
@@ -236,6 +258,10 @@ module.exports.register = function() {
     const pipelineAnnotations = helpers.processAnnotationsData(rawAnnotationsData, "policy.pipeline")
     const releaseAnnotations = helpers.processAnnotationsData(rawAnnotationsData, "policy.release")
```

Comment:
Should probably be a loop over the discovered package namespaces instead of hard coding "release" and "pipeline", but that's a refactor for the future.